third_party/BUILD

```diff
@@ -50,8 +48,6 @@ filegroup(
 filegroup(
     name = "gpl-srcs",
     srcs = [
-        "//third_party/checker_framework_dataflow:srcs",
```

Comment:
Since you started this, can you also add them to the java tools dist archive? It's defined by `src:java_tools_dist_java*`

Reply:
Done.